nengo/spa/assoc_mem.py

```diff
@@ -100,18 +95,22 @@ def __init__(self, input_vocab, output_vocab=None,  # noqa: C901
             if wta_output:
                 self.am.add_wta_network(wta_inhibit_scale, wta_synapse)
 
-            if threshold_output:
-                self.am.add_threshold_to_outputs()
+            if cleanup_output:
+                self.am.add_cleanup_output(
+                    replace_output=replace_output_with_cleaned_output)
 
             self.input = self.am.input
             self.output = self.am.output
 
+            if cleanup_output and not replace_output_with_cleaned_output:
+                self.cleaned_output = self.am.cleaned_output
+
             if inhibitable:
                 self.inhibit = self.am.inhibit
 
-            self.utilities = self.am.utilities
-            if threshold_output:
-                self.thresholded_utilities = self.am.thresholded_utilities
+            self.utilities = self.am.output_utilities
```

Comment:
Why is it called `utilities` in the module but `output_utilities` in the network?